INSTALLATION.rst

```diff
@@ -303,25 +322,10 @@ using :term:`pip` run a somewhat elevated risk of unmet dependencies as the
 independent codes evolve. Stable releases of :term:`FiPy` are packaged for
 distribution through :term:`PyPI`.
 
-.. _ANACONDA:
-
-Continuum Analytics Anaconda
-----------------------------
-
-http://continuum.io/anaconda
-
-In addition to the scientific :term:`Python` stack, the Anaconda package manager
-also provides virtual environment management. Keeping separate installations is useful
-*e.g.* for comparing :term:`Python` 2 and :term:`Python` 3 software stacks, or
-when the user does not have sufficient privileges to install software system-wide.
-
-.. attention::
-
-   :term:`Pysparse` and :term:`FiPy` are not presently included in Anaconda,
-   so you will need to separately install them manually.
-
 .. _ECP:
 
+.. _EPD:
+
 Enthought Canopy
```

Comment:
Shall we delete the Enthought Canopy links?

The Python(x,y) links should definitely be deleted. It doesn't look well maintained.

Reply:
Should the "Scientific Python Packages" section go altogether? The Canopy and Python (x, y) sections are dated and the Contiuum Analytics should probably be merged with the Recommended Method as that is explaining how to install with Conda. Also, the entire Recommend Method is one long Attention box. I don't think that is intended.

Reply:
I see, you've already cleaned loads of this up. It's much better now. Maybe just think about my above comment as well. It's definitely easier to read when it's shorter.

Reply:
My intention is to address all of this with a separate pull request (although, I already addressed some of it in the course of making the `conda` section accurate.) Without merging this pull, I can't do any other work because the tests don't pass on `develop`, so I urge not letting these edits derail this pull request any further.